tools/ci_build/build.py

```diff
@@ -268,7 +268,7 @@ def check_md5(filename, expected_md5):
 
```

Comment:
As you can see, in github it shows you changed the file mode from 100755 to 100644. That's why the Android build is failing. Please fix it.

Reply:
Thanks